relay-protocol/src/condition.rs

```diff
@@ -163,7 +163,7 @@ pub struct GlobCondition {
     /// A list of glob patterns to check.
     ///
     /// Note that this cannot be a single value, it must be a list of values.
-    pub value: TypedPatterns<CaseInsensitive>,
+    pub value: TypedPatterns,
```

Comment:
We could introduce an alias like `DefaultPatterns` for `TypedPatterns<DefaultPatternConfig>`. Having just `TypedPatterns` reads a bit oddly to me.